optimade/models/toplevel.py

```diff
@@ -185,11 +185,13 @@ class InfoResponse(Success):
 class EntryResponseOne(Success):
     meta: ResponseMeta = Schema(...)
     data: Union[EntryResource, Dict[str, Any], None] = Schema(...)
+    included: Optional[List[Dict[str, Any]]] = Schema(...)
```

Comment:
These should be identical to `data` for `EntryResponseMany`, and also be added to the specific entry responses.
Of course keeping the `Optional`.

Reply:
Yep, it will be once we're getting the actual references out, rather the than returning the relationship again here

Reply:
This can be updated now, right?

Reply:
Quite right, I've updated it in my last commit, but unlike `data` I've made it optional as per the spec